docs/life_of_a_dataset.md

```diff
@@ -21,7 +21,11 @@ The statistical variable DCIDs should be human-readable, encapsulating the
 meaning of its triples.  The naming rules are summarized in [this
 doc](https://docs.google.com/document/d/1YtmQu_gOXENt4W2lmlK-8Mtnw_XB3kZlLBjSN-gT0oE/preview#bookmark=kix.bdksa8hbv3si).
 
-When the variables are finalized, they get checked into the [schema
+You'll need approval from the Data Commons team on any new variables (see the 
+"Defining location and schema mapping" section 
+[here](https://github.com/datacommonsorg/data/blob/master/docs/README.md)). 
+
+When the variables are finalized, they get checked into the [schema 
```

Comment:
This doc doesn't talk about the doc at all.  Should we add a section before "## Define SVs", that reiterate the need to go through the doc review to finalize mapping / SVs?

Reply:
Sg, to confirm you mean add a section to the README.md doc, right?

Reply:
added a new section for Import Design Review. Once I'm done updating the schema-less guide, we can link to that here too. For now, this should be sufficient!